texts/formatted_harry_potter.txt

```diff
@@ -0,0 +1,1679 @@
+	"," cm
+"<Harry>"
+	"Harry" np ant m sg
+"<Potter>"
+;	"potter" n sg
+	"Potter" np cog sg
+"<,>"
+	"," cm
+"<and>"
+	"and" cnjcoo
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
+"<friends>"
+	"friend" n pl
+"<Hermione>"
+	"Hermione" np ant f sg
+"<Granger>"
+	"Granger" np cog sg
+"<and>"
+	"and" cnjcoo
+"<Ron>"
+	"Ron" np ant m sg
+"<Weasley>"
+	"Weasley" np cog sg
+"<,>"
+	"," cm
+"<all>"
+;	"all" adv
+;	"all" predet sp
+	"all" det qnt pl
+;	"all" prn qnt mf sp
+"<of>"
+	"of" pr
+"<whom>"
+	"whom" prn rel an mf sp
+"<are>"
+	"be" vbser pres
+"<students>"
+	"student" n pl
+"<at>"
+	"at" pr
+"<Hogwarts>"
+	"Hogwarts" np top sg
+"<School>"
+;	"school" adj
+	"school" n sg
+"<of>"
+	"of" pr
+"<Witchcraft>"
+	"witchcraft" n sg
+"<and>"
+	"and" cnjcoo
+"<Wizardry>"
+	"*Wizardry"
+"<.>"
+	"." sent
+"<The>"
+	"the" det def sp
+"<main>"
+	"main" adj
+"<story>"
+	"story" n sg
+"<arc>"
+	"arc" n sg
+"<concerns>"
+;	"concern" n pl
+	"concern" vblex pres p3 sg
+"<Harry>"
+	"Harry" np ant m sg
+"<'s>"
+	"'s" gen
+;	"be" vbser pres p3 sg
+;	"have" vbhaver pres p3 sg
+"<struggle>"
+;	"struggle" n sg
+	"struggle" vblex inf
+;	"struggle" vblex pres
+;	"struggle" vblex imp
+"<against>"
+	"against" adj
+	"against" adv
+	"against" pr
+"<Lord>"
+	"lord" n sg
+;	"lord" vblex inf
+;	"lord" vblex pres
+;	"lord" vblex imp
+;	"Lord" np cog sg
+"<Voldemort>"
+	"Voldemort" np cog sg
+;	"Voldemort" np ant m sg
+"<,>"
+	"," cm
+"<a>"
+	"a" det ind sg
+"<dark>"
+;	"dark" n sg
+	"dark" adj sint
+"<wizard>"
+	"wizard" n sg
+"<who>"
+;	"who" prn itg mf sp
+	"who" prn rel an mf sp
+"<intends>"
+	"intend" vblex pres p3 sg
+"<to>"
+	"to" pr
+"<become>"
+	"become" vblex inf
+;	"become" vblex pres
+;	"become" vblex pp
+;	"become" vblex imp
+"<immortal>"
+	"immortal" adj
+"<,>"
+	"," cm
+"<overthrow>"
+;	"overthrow" n sg
+	"overthrow" vblex inf
+;	"overthrow" vblex pres
+;	"overthrow" vblex imp
+"<the>"
+	"the" det def sp
+"<wizard>"
+	"wizard" n sg
+"<governing>"
+	"governing" adj
+;	"govern" vblex pprs
+;	"govern" vblex subs
+;	"govern" vblex ger
+"<body>"
+	"body" n sg
+"<known>"
+;	"known" adj
+	"know" vblex pp
+"<as>"
+;	"as" adv
+;	"as" preadv
+;	"as" cnjsub
+;	"as" cnjadv
+	"as" pr
+"<the>"
+	"the" det def sp
+"<Ministry>"
+	"ministry" n sg
+"<of>"
+	"of" pr
+"<Magic>"
+;	"magic" adj
+	"magic" n sg
+"<and>"
+	"and" cnjcoo
+"<subjugate>"
+	"subjugate" vblex inf
+;	"subjugate" vblex pres
+;	"subjugate" vblex imp
+"<all>"
+;	"all" adv
+;	"all" predet sp
+	"all" det qnt pl
+;	"all" prn qnt mf sp
+"<wizards>"
+	"wizard" n pl
+"<and>"
+	"and" cnjcoo
+"<Muggles>"
+	"*Muggles"
+"<(>"
+	"(" lpar
+"<non>"
+	"non" adv
+"<->"
+	"-" guio
+"<magical>"
+	"magical" adj
+"<people>"
+;	"people" n sg
+	"person" n pl
+"<)>"
+	")" rpar
+"<.>"
+	"." sent
+"<Since>"
+;	"since" adv
+;	"since" cnjsub
+	"since" pr
+"<the>"
+	"the" det def sp
+"<release>"
+;	"release" n sg
+	"release" vblex inf
+;	"release" vblex pres
+;	"release" vblex imp
```

Comment:
after determiner and before preposition, should be noun

Reply:
Oops these both i noticed as Nouns. But missed.